OpenRA.Game/Network/Order.cs

```diff
@@ -147,16 +151,30 @@ public static Order Deserialize(World world, BinaryReader r)
 
 						var targetString = flags.HasField(OrderFields.TargetString) ? r.ReadString() : null;
 						var queued = flags.HasField(OrderFields.Queued);
+
+						List<Actor> extraActors = null;
+						if (flags.HasField(OrderFields.ExtraActors))
+						{
+							var count = r.ReadInt32();
+							extraActors = new List<Actor>(count);
+							for (int i = 0; i < count; i++)
```

Comment:
`var i`

Reply:
Do we have a reason for preferring a list over an array?

If not, we could simplify this down to something like
```csharp
if (world != null)
    extraActors = Exts.MakeArray(count, id => world.GetActorById(actorId));
else
   r.ReadBytes(4*count);
```

Reply:
No particular reason. Changed to array.